integrations/ms-teams/index.md

```diff
@@ -0,0 +1,102 @@
+# Prepare zip archive for MS Teams application
+
+1. Create a folder with an application name
+2. Put on the folder (example file attached to the task):
+    1. **color.png** file which contains program icon file (280x280pt default). This icon will be shown on the MS Teams tab panel;
+    2. **outline.png** file which contains the color that will be considered transparent on the icon. Usually a 32x32 pixel picture with a white fill;
+    3. **manifest.json** file with an application tab presence.
+3. If the application is a new one it needs registration and gets a unique id - [app registration](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade).
+4. Full manifest file description available on [ms teams developers documentation](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema).
+5. Minimum content required on manifest.json file:
```

Comment:
Let's wrap manifest.json in code block markup so the file name stands out more - `manifest.json`.
Do it please for all `manifest.json` file mentions.